### PR TITLE
TLS early data: logging updates

### PR DESCRIPTION
**Pull request by bneradt**
This patch updates TLS logging by making TLS hook names human readable and adds some early data SNI action logs.

### CODE REVIEW

File: iocore/net/SSLNetVConnection.cc [hunk 1]
```diff
@@ -2511,7 +2545,9 @@ SSLNetVConnection::update_early_data_config(uint32_t max_early_data, uint32_t re
 #else
     // If SSL_set_max_early_data is unavailable, it's probably BoringSSL,
     // and SSL_set_early_data_enabled should be available.
-    SSL_set_early_data_enabled(ssl, max_early_data > 0 ? 1 : 0);
+    bool const early_data_enabled = max_early_data > 0 ? 1 : 0;
+    SSL_set_early_data_enabled(ssl, early_data_enabled);
+    Debug("ssl", "Called SSL_set_early_data_enabled with %d", early_data_enabled);
     Warning("max_early_data is not used due to library limitations");
```
Comment (bneradt) [inline iocore/net/SSLNetVConnection.cc:2551]: @duke8253 : Should this warning be changed? Or removed? Maybe the intention here is that with boringssl there is no way to explicitly turn off replay (see line 2543 above).

Comment (duke8253) [reply iocore/net/SSLNetVConnection.cc:2551]: Yes, I think this can be removed here.

**Comment by bneradt** [comment]
[approve ci autest]

**Comment by bneradt** [comment]
[approve ci autest]

File: iocore/net/SSLNetVConnection.cc (hunk 1 above)
Comment (bneradt) [reply iocore/net/SSLNetVConnection.cc]: Thanks. Removed.

**Review by duke8253 — DISMISSED** [review]
Looks good, still needs someone not from yahoo to approve though.

**Review by randall — APPROVED** [review]

**Review by randall — APPROVED** [review]

**Comment by bneradt** [comment]
[approve ci rocky freebsd osx]

**Comment by bneradt** [comment]
[approve ci rocky]

**Comment by bneradt** [comment]
[approve ci rocky]

**Pull request merged**